kit/controls/src/UIActionButton/hooks.ts

```diff
@@ -28,9 +28,9 @@ export const useButtonColorScheme = (type: UIActionButtonType): ActionButtonColo
                         disabled: ColorVariants.BackgroundNeutral,
                     },
                     content: {
-                        normal: ColorVariants.TextPrimary,
-                        hover: ColorVariants.TextPrimary,
-                        pressed: ColorVariants.TextPrimary,
+                        normal: ColorVariants.TextPrimaryInverted,
+                        hover: ColorVariants.TextPrimaryInverted,
+                        pressed: ColorVariants.TextPrimaryInverted,
```

Comment:
Why `TextPrimaryInverted `? According to the layouts should be `TextPrimary `
![image](https://user-images.githubusercontent.com/36547799/146734124-82111859-2fb0-4220-a16c-2a9ab7cf7e41.png)

Reply:
It was black for light theme :)

Reply:
<img width="352" alt="Screenshot_2021-12-18_at_20 09 35" src="https://user-images.githubusercontent.com/6286454/146736451-a57f5863-1f5f-40f0-83c2-4b70495d18ff.png">

Reply:
Why do you think it's wrong? If so, then you need to ask to change the color on the layouts.

Reply:
It will be black with `TextPrimaryInverted` with dark theme instead

Reply:
IDK, for me it's obvious that it's wrong 😂

Reply:
changed)

Reply:
`StaticTextPrimaryLight` is ok for me. Can you ask @eteslov to change the color on the layouts?